pkg/lambda/grpc/config/config.go

```diff
@@ -99,16 +99,16 @@ func GetConnectorConfigServiceClient(ctx context.Context, clientID string, clien
 
 	dialOpts := []grpc.DialOption{
 		grpc.WithTransportCredentials(transportCreds),
-		grpc.WithUserAgent(fmt.Sprintf("%s baton-lambda/%s", clientName, "v0.0.1")),
+		grpc.WithUserAgent(fmt.Sprintf("%s baton-lambda/%s", clientID, "v0.0.1")),
```

Comment:
This version should probably be in a constant somewhere. Or maybe it should be the baton-sdk version?

Reply:
A const prefix would be nice at least.  I think we want the installation domain (dev instance/staging/prod) in the UA because we commingle lambdas in the same aws account for all dev instances and they will send traffic through the same proxy.